addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -58,9 +58,9 @@ options.registry.mailing_list_subscribe = options.Class.extend({
      * @see this.selectClass for parameters
      */
     toggleThanksMessage(previewMode, widgetValue, params) {
```

Comment:
The message is not a button anymore, but the option is still named "Display Thanks Button".
![image](https://github.com/user-attachments/assets/10fad8da-01bf-48b0-ada8-d2617ea1dce8)

If the form replacement by a thanks message is general - maybe that toggle should also become available for Form mode ? (Maybe to be confirmed with a PO)

Reply:
I've updated the toggle name 👍 

> If the form replacement by a thanks message is general - maybe that toggle should also become available for Form mode ? (Maybe to be confirmed with a PO)

Since this PR is about redesigning the snippet, I think we could address this point in another technical PR?

Reply:
Yes indeed, I'll create a task on our side.